src/hooks/useDoNotPromptForParty.ts

```diff
@@ -0,0 +1,34 @@
+import { useAppSelector } from 'src/hooks/useAppSelector';
+import { useInstanceIdParams } from 'src/hooks/useInstanceIdParams';
+import type { IAltinnWindow } from 'src/types';
+
+export function useDoNotPromptForParty(): boolean | null {
```

Comment:
I don't quite like the double negative (if `doNotPromptForParty === false` it means to prompt for party), but I guess it's closer to the profile setting.

Reply:
I know, this has been painful to wrap my head around the whole time, but the setting is literally called `doNotPromptForParty`, and `false` means prompt for party every time. 😅

Reply:
I guess I could change the hook to be easier to understand